core-experimental/include/iDynTree/Core/PositionRaw.h

```diff
@@ -8,86 +8,47 @@
 #ifndef IDYNTREE_POSITION_RAW_H
 #define IDYNTREE_POSITION_RAW_H
 
-#include "IVector.h"
+#include <iDynTree/Core/IVector.h>
+#include <iDynTree/Core/VectorFixSize.h>
 #include <string>
 
 namespace iDynTree
 {
     class RotationRaw;
-    
+
     /**
      * Class providing the raw coordinates for iDynTree::Position class.
      *
      * \ingroup iDynTreeCore
      */
-    class PositionRaw: public IVector
+    class PositionRaw: public Vector3
```

Comment:
Being able to access the elements of the vector was one of the core requirements, to actually *use* the data after any kind of computation.